integration/kubernetes/singleMaster-hdfsJournal/worker/alluxio-worker-daemonset.yaml.template

```diff
@@ -84,9 +84,10 @@ spec:
           volumeMounts:
             - name: alluxio-domain
               mountPath: /opt/domain
+            
             - name: secret-alluxio-hdfs-config-volume
               mountPath: /secrets/hdfsConfig
-              readOnly: true
+              readOnly: true            
```

Comment:
can we get rid of the extra space?

Reply:
Unfortunately these extra whitespaces are more tricky than I thought, due to the levels of templating. I'll spend more time looking into this.